src/main/java/core/basesyntax/service/GetStrategyService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.util.Operation;
+
+public interface GetStrategyService {
+    OperationHandler getStrategyService(Operation operation);
```

Comment:
```suggestion
    OperationHandler get(Operation operation);
```
your method doesn't return StrategyService but OperationHandler